packages/atomic/src/components/common/radio-button.tsx

```diff
@@ -0,0 +1,55 @@
+import {FunctionalComponent, h} from '@stencil/core';
+import {createRipple} from '../../utils/ripple';
+import {
+  ButtonStyle,
+  getClassNameForButtonStyle,
+  getRippleColorForButtonStyle,
+} from './button-style';
+
+export interface RadioButtonProps {
+  style: ButtonStyle;
+  groupName: string;
```

Comment:
I decided to make `groupName` non-optional, since IMO every radio button should be related to other radio buttons.